### PR TITLE
Update causal Conv1D doc : uses zero padding #8751

### PR DESCRIPTION
**Pull request by BertrandDechoux**
### Summary

Update  Conv1D documentation about causal padding : it uses zero padding.

### Related Issues

See issue #8751

### PR Overview

- [n] This PR requires new unit tests [y/n] (make sure tests are included)
- [y] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [y] This PR is backwards compatible [y/n]
- [n] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Comment by BertrandDechoux** [comment]
**Note :** Do I need to do something about "make sure the docs are up-to-date" ? Generate a new version ?

**Comment by gabrieldemarmiesse** [comment]
This is for example if you were to add a new feature to keras. A lot of people add the feature then forget to change the corresponding sections in the docs. In your case you are fine.

**Comment by BertrandDechoux** [comment]
You are welcome. I was using Keras in order to generate a wavenet like model. So I have wondered about that part of the documentation.
